setup.py

```diff
@@ -24,7 +24,7 @@ def readme():
     'google-auth>=1.0.0',
     'google-auth-httplib2>=0.0.1',
     'google-auth-oauthlib>=0.0.1',
-    'google-cloud-bigquery>=0.25.0,<=0.26.0',
+    'google-cloud-bigquery==0.26.0',
```

Comment:
I have a strong but proprietary (i.e. not everyone agrees with me) that hard pins shouldn't be in `setup.py`. If they're here then this will downgrade to `0.26.0` whenever this library is installed. 

It's OK to have them in `requirements.txt` because that doesn't force behavior.

Reply:
0.26.0 is the latest so at the moment it would be a forced upgrade if anything (maybe I'll put 0.25.0 as an acceptable version back in so less users are forced to do so), but as others have mentioned, the major concern is that Google makes no guarantee about future behavior. At least until Google indicates that google-cloud-bigquery can be depended on for stability (at least with core functionality), I lean toward hard pinning here as well so a user always is never surprised by a broken product due to breaking changes in future versions. But happy to remove if others agree.

Reply:
Good points and I agree there's a tradeoff

@jreback thoughts / verdict?

Reply:
I know for a fact that `google-cloud[-bigquery]` will continue to make breaking changes, as I've been responsible for quite a few that will make it into later versions (0.28 or maybe 0.29).

Since `pandas-gbq` isn't currently getting built at HEAD to track the `google-cloud` changes, we won't be able to catch the breaking changes here before they release, so it's probably better to pin.

The alternative is to add some CI like we have to track Pandas changes at HEAD.